simulation_parameters/microbe_stage/compounds.json

```diff
@@ -31,47 +17,36 @@
       "g": 0.39,
       "b": 0.0
     },
-    "IconPath": "res://assets/textures/gui/bevel/ammonia.png"
+    "IconPath": "res://assets/textures/gui/bevel/Ammonia.svg"
   },
-  "phosphates": {
-    "Name": "PHOSPHATE",
+  "atp": {
```

Comment:
Could you keep the same order in this file that there was previously? That'd make it much easier to see what was actually changed. (even if technically it doesn't impact anything in the game)

Reply:
I thought it would be worth alphabetizing the compounds, but I see your point.

Reply:
I guess that could theoretically help, but the file is so long that it is basically the only good way to navigate it is to use "search forward" feature in a text editor.